### PR TITLE
Productpage

### PR DESCRIPTION
**Pull request by ckahub**
#filter 
We have to create toogle function in checkbox for filterby categories and brands
for filter by price sort function is used
###Filterbyprice
###Filterbycategories
###Fillterbybrands
###FilterbypriceRange

https://user-images.githubusercontent.com/7824010/164950981-f99dea2b-3a73-4398-84ee-188b9194c754.mp4

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,35 +1,34 @@
...
-import {LoginIN} from "./pages/Login/LoginIN";
-import {Register} from "./pages/Register/Register";
-import {Cart} from "./pages/Cart/Cart";
-import {Link,Routes,Route} from 'react-router-dom';
+import { Homepage } from "./pages/Homepage/Homepage";
+import { Productpage } from "./pages/Productpage/Productpage";
+import { Wishlist } from "./pages/Wishlist/Wishlist";
+import { LoginIN } from "./pages/Login/LoginIN";
+import { Register } from "./pages/Register/Register";
+import { Cart } from "./pages/Cart/Cart";
```
Comment (dusariudaykumar) [inline src/App.js:9]: You can use an ``` index.js``` file and add all your exports here and you can simply import all your component from a single file.

File: src/Reducer/sortReducer.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    //   };
+    // case "BRAND":
+    //   return {
+    //     ...state,
+    //     brand: {
+    //       ...state.brand,
+    //       [action.payload.toLowerCase()]:
+    //         !state.brand[action.payload.toLowerCase()] || false,
+    //     },
+    //   };
```
Comment (dusariudaykumar) [inline src/Reducer/sortReducer.js:115]: remove unused code

File: src/pages/Productpage/Productpage.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
+/* .container{
+    display: flex;
+    flex-wrap: wrap;
+    height: 100vh;
+    width: 100vw;
+} */
```
Comment (dusariudaykumar) [inline src/pages/Productpage/Productpage.css:6]: same here

**Review by dusariudaykumar — APPROVED** [review]
The code is clean and great folder structure.
Just some neat picks are there, have a look at them.

**Review by shashikantmaurya0007 — APPROVED** [review]
Remove the commented code.create one index.js for import .rest looks good

File: src/Reducer/sortReducer.js (hunk 2 above)
Comment (ckahub) [reply src/Reducer/sortReducer.js:115]: done with this

**Comment by ckahub** [comment]
done all changes

**Pull request merged**